### ISSUE TITLE
Warnings counts twice as issues

### ISSUE THREAD

**Issue by geret1** [FlutterFlow/flutterflow-issues#3451]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Mismatch between issues numbers and warnings plus errors

### Expected Behavior

Consistent figures representations.

### Steps to Reproduce

1. Have a random project.
2. Triage issues window.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

N/A

### Visual documentation

<img width="288" alt="Screenshot 2024-07-22 at 08 58 49" src="https://github.com/user-attachments/assets/579229d6-9ab7-457a-be75-50b3478ed0e9">

### Environment

```markdown
- FlutterFlow version: v4.1.74+ released July 18, 2024
- Platform:simulator
- Browser name and version: chrome
- Operating system and version affected: N/A
```

**Comment by msusviela** [comment]
Hi @geret1. I was not able to reproduce this issue. Could you please provide more information on the issue? I'll need to know the message for the warning and also I'll need a bug report code so I can look further into the issue. 

This guide might help you to create the bug report code: [https://docs.flutterflow.io/troubleshooting/submit-bug-reports](https://docs.flutterflow.io/troubleshooting/submit-bug-reports)

Thanks!

**Comment by geret1** [comment]
Hi @msusviela! but in this case the error doesn't come from a specific component but the main FlutterFlow toolbar so, afaik, it's not possible to get the bug report code. Let me know if I can help you in advanced!

**Comment by msusviela** [comment]
From our side we were not able to reproduce it, all the warnings we were getting were counting only once, so it seems specific to that type of warning. With the bug code we can access to your errors and warnings as well. Without knowing what's going on, we won't be able to replicate it

**Comment by msusviela** [comment]
Closing this issue since we haven't heard back from you. If you are still facing the problem please submit a new issue. Have a great day!
